frontend/public/components/monitoring/dashboards/index.tsx

```diff
@@ -261,14 +266,9 @@ const TimespanDropdown_: React.FC<TimespanDropdownProps> = ({ timespan, setTimes
   );
 };
 
-export const TimespanDropdown = connect(
-  ({ UI }: RootState) => ({
-    timespan: UI.getIn(['monitoringDashboards', 'timespan']),
-  }),
-  {
-    setTimespan: UIActions.monitoringDashboardsSetTimespan,
-  },
-)(TimespanDropdown_);
+export const TimespanDropdown = connect(({ UI }: RootState) => ({
+  timespan: UI.getIn(['monitoringDashboards', 'timespan']),
+}))(TimespanDropdown_);
```

Comment:
IMO, if we're going to touch this, we might as well just get rid of `connect` here and switch to the `useSelector` Redux hook.

Reply:
👍 I'll address this in a follow-up PR. Thanks!